### PR TITLE
Print link to debugger UI

### PR DESCRIPTION
**Pull request by bjchambers**
After successfully registering a debuggee, print a link to go to the
appropriate UI.

### CODE REVIEW

**Comment by bjchambers** [comment]
R: @davorbonaci

File: sdk/src/main/java/com/google/cloud/dataflow/sdk/runners/DataflowPipelineRunner.java [hunk 1]
```diff
@@ -424,6 +424,12 @@ public static DataflowPipelineRunner fromOptions(PipelineOptions options) {
     return super.apply(new AssignWindows<>(transform), input);
   }
 
+  private String debuggerMessage(String projectId, String uniqifier) {
```
Comment (davorbonaci) [inline sdk/src/main/java/com/google/cloud/dataflow/sdk/runners/DataflowPipelineRunner.java]: typo: uniquifier

File: sdk/src/main/java/com/google/cloud/dataflow/sdk/runners/DataflowPipelineRunner.java [hunk 2]
```diff
@@ -424,6 +424,12 @@ public static DataflowPipelineRunner fromOptions(PipelineOptions options) {
...
+    return String.format("To view the debugger console, go to "
```
Comment (davorbonaci) [inline sdk/src/main/java/com/google/cloud/dataflow/sdk/runners/DataflowPipelineRunner.java]: Branding issue here. For example: "To debug your job, visit Google Cloud Debugger at: ".

From their website: "Visit the Debug page of the Cloud Platform Console to use Google Cloud Debugger."

Comment (davorbonaci) [inline sdk/src/main/java/com/google/cloud/dataflow/sdk/runners/DataflowPipelineRunner.java]: Branding issue here. For example: "To debug your job, visit Google Cloud Debugger at: ".

From their website: "Visit the Debug page of the Cloud Platform Console to use Google Cloud Debugger."

**Comment by davorbonaci** [comment]
LGTM.

File: sdk/src/main/java/com/google/cloud/dataflow/sdk/runners/DataflowPipelineRunner.java (hunk 1 above)
Comment (bjchambers) [inline sdk/src/main/java/com/google/cloud/dataflow/sdk/runners/DataflowPipelineRunner.java]: Done.

File: sdk/src/main/java/com/google/cloud/dataflow/sdk/runners/DataflowPipelineRunner.java (hunk 2 above)
Comment (bjchambers) [inline sdk/src/main/java/com/google/cloud/dataflow/sdk/runners/DataflowPipelineRunner.java]: Done.

**Pull request closed without merging**